app/views/settings/show.html.haml

```diff
@@ -24,6 +24,21 @@
       %section
         = render ReactComponents::Settings::HandleForm.new
 
+      %section.flex.flex-col.items-start
+        %h2 GitHub Integration
+        %hr.c-divider.--small
+        - if current_user.provider.present?
+          %p.text-16.leading-140
+            You Exercism and GitHub accounts are linked. Any pull requests that are merged on GitHub should award you reputation on your Exercism account.
```

Comment:
Just a thought: maybe link to https://exercism.org/docs/using/product/reputation for details on what reputation is and how you get it?